### PR TITLE
Add page for v1.4 status endpoint as well as version history section on README.md

### PR DESCRIPTION
**Pull request by poisa**
- Add page for v1.4 status endpoint.
- Marked v1.3 status endpoint as deprecated.
- Add version history to README.md

### CODE REVIEW

**Review by oikeme — APPROVED** [review]
LGTM

File: jetpack/jetpack-start-endpoints/determining-provisioning-status-v1.4.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+## Response Parameters
+
+- __blog__:                                (array) Blog specific information.
+  - __is_connected__:                      (bool) Is the site connected to WordPress.com?
+  - __has_backups_credentials__:           (bool) Does the site have Jetpack Backup SSH credentials set?
+- __products__:                            (array) A list with everything that was provisioned for this site.
+  - __slug__:                              (string) The slug of the provisioned product.
+  - __is_provisioned_by_partner__:         (bool) Was this product provisioned by a partner?
+  - __is_provisioned_by_calling_partner__: (bool) Was this product provisioned by the partner making this status request?
+  - __is_pending__:                        (bool) Does the site have a pending product that requires connection authorization?
```
Comment (ebinnion) [inline jetpack/jetpack-start-endpoints/determining-provisioning-status-v1.4.md]: Slight language change:

"(bool) Is this product pending connection authorization before being provisioned?"

**Review by ebinnion — COMMENTED** [review]
I left one minor comment about clarifying some language. But, it LGTM besides that.

File: jetpack/jetpack-start-endpoints/determining-provisioning-status-v1.4.md (hunk 1 above)
Comment (poisa) [reply jetpack/jetpack-start-endpoints/determining-provisioning-status-v1.4.md]: Thanks. Fixed.

**Pull request merged**